library/Fission/Mail/SendInBlue/ApiKey/Types.hs

```diff
@@ -0,0 +1,14 @@
+module Fission.Mail.SendInBlue.ApiKey.Types (ApiKey(..)) where
+
+import Fission.Prelude
+
+newtype ApiKey = ApiKey { unApiKey :: Text }
+  deriving          ( Generic )
```

Comment:
Why do we need `Generic` here?

Reply:
:man_shrugging: 
i'll be honest I'm not really sure what `Generic` does. I've just been throwing it on `newtype`s that are easily derivable from primitive-ish values :sweat_smile: